app/helpers/admin/questions_helper.rb

```diff
@@ -0,0 +1,15 @@
+module Admin::QuestionsHelper
+	def bar_param(q_level)
+    width = '100%'
+    color = 'danger'
+    if q_level == 'beginner'
```

Comment:
refactor this by using hint:
```ruby
a,b = b,a
```
contact me via Skype if you have any questions regarding this.